anvil/anvil.go

```diff
@@ -36,7 +36,7 @@ var (
 )
 
 const (
-	host                 = "127.0.0.1"
+	defaultHost          = "127.0.0.1"
```

Comment:
this can be removed, since host will default to "127.0.0.1" based on the default CLI flag value